proof/crefine/ARM/IsolatedThreadAction.thy

```diff
@@ -673,11 +685,11 @@ lemma transferCaps_simple_rewrite:
    (transferCaps mi caps ep r rBuf)
    (return (mi \<lparr> msgExtraCaps := 0, msgCapsUnwrapped := 0 \<rparr>))"
   including no_pre
+  supply empty_fail_getReceiveSlots[wp] (* FIXME *)
```

Comment:
Just confirming, is the FIXME here that `empty_fail_getReceiveSlots` should be globally `wp`?

Reply:
That is correct